File: Mobile/app/src/main/java/com/um/feri/direct4me/android/api/model/login/LoginResponse.kt
package com.um.feri.direct4me.android.api.model.login

import java.util.Date

data class LoginResponse(
    val guid: String,
    val email: String,
    val fullname: String,
    val totalLogins: Int,
    val lastAccess: Date,
    val isFaceUnlockEnabled: Boolean
)

File: Web/Direct4Me.Minimal.Api/Models/Login/LoginResponse.cs
using System.ComponentModel.DataAnnotations;

namespace Direct4Me.Minimal.Api.Models.Login;

public record LoginResponse(
    [Required] string Guid,
    [Required] string Email,
    [Required] string Fullname,
    [Required] int TotalLogins,
    [Required] DateTime LastAccess,
    [Required] bool IsFaceUnlockEnabled
);